lib/api/src/grpc/models.rs

```diff
@@ -43,6 +43,14 @@ pub struct ApiResponse<D> {
     pub result: Option<D>,
     pub status: ApiStatus,
     pub time: f64,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub hardware_usage: Option<HardwareUsage>,
+}
+
+#[derive(Debug, Serialize, JsonSchema)]
```

Comment:
Not for this PR, but it looks like misplacement of models. Since models implement `JsonSchema`, they shouldn't be in grpc subfolder